src/methods/ReportTemplateGenerator.tsx

```diff
@@ -146,4 +168,17 @@ class ReportInfo {
   private deadlineElement = this.tdElements[2]
 }
 
-export { ReportTemplateGenerator }
+const RowContent = ({
+  header,
+  data,
+}: {
+  header: string
+  data: JSX.Element | string
```

Comment:
I believe `ReactElement` is better as long as we use React.

Reply:
3c8c511361d8e241ba33c2645b1bbcc4fd67672f resolved this review.